src/app/login-form.tsx

```diff
@@ -0,0 +1,140 @@
+"use client"
+
+import { Auth } from "@supabase/auth-ui-react"
+import {
+  Session,
+  createClientComponentClient,
+} from "@supabase/auth-helpers-nextjs"
+import { Database } from "../types/supabase"
+import { useState } from "react"
+import { redirect, useRouter } from "next/navigation"
+import Link from "next/link"
+import Button, { button } from "@/components/common/Button"
+import Input, { input } from "@/components/common/Input"
+import { css, cx } from "../../styled-system/css"
+import Separator from "@/components/common/Separator"
+
+export default function LoginForm({ session }: { session: Session | null }) {
+  const supabase = createClientComponentClient<Database>()
+  const router = useRouter()
+
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [isMagicLinkLogin, setIsMagicLinkLogin] = useState(false)
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault()
+    const { error } = await supabase.auth.signInWithPassword({
+      email,
+      password,
+    })
+    if (error) {
```

Comment:
bug: I can't login with credentials, only with the magic link. Is it the same for you? 🤔

Reply:
I am able to login hehe, maybe we can try on our call and see what's going on